src/Compilers/Core/Portable/Operations/DisposeOperationInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#nullable enable
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    internal struct DisposeOperationInfo
```

Comment:
Feels like this file should be located in the same place as `ForEachEnumeratorInfo`.